packages/xo-web/src/xo-app/settings/acls/index.js

```diff
@@ -213,7 +213,7 @@ export default class Acls extends Component {
       this.setState({
         subjects: [],
         objects: [],
-        action: '',
+        action: undefined,
```

Comment:
Why is it initialized to `null` in the constructor?

Reply:
I have no idea. I think it should be `undefined`.

Reply:
Also, we check against `undefined` [here](https://github.com/vatesfr/xen-orchestra/blob/pierre-fix-acls-action/packages/xo-web/src/xo-app/settings/acls/index.js#L119).